src/main/java/com/alphawallet/attestation/NFTAttestation.java

```diff
@@ -0,0 +1,35 @@
+package com.alphawallet.attestation;
+
+import com.alphawallet.attestation.core.ASNEncodable;
+import com.alphawallet.attestation.core.Validateable;
+import com.alphawallet.ethereum.ERC721Token;
+import org.bouncycastle.asn1.ASN1EncodableVector;
+import org.bouncycastle.asn1.ASN1Primitive;
+import org.bouncycastle.asn1.ASN1Sequence;
+import org.bouncycastle.asn1.DERSequence;
+
+public class NFTAttestation implements ASNEncodable, Validateable {
+    private final SignedAttestation att;
+    private final ASN1Sequence token;
+
+    public NFTAttestation(SignedAttestation att, ERC721Token nftToken) {
```

Comment:
I think we still need the optional autograph as a digest value

Reply:
What is the Autograph in this case? Previously we were going to include the digest of the .png of the actual autograph but now it's a procedurally added Twitter name.

Reply:
On Friday @colourful-land mentioned that it would be a hash digest of the transmogrified  object.